packages/desktopjs-electron/src/electron.ts

```diff
@@ -4,29 +4,50 @@
  */
 
 import {
-    registerContainer, ContainerWindow, PersistedWindowLayout, Rectangle, Container, WebContainerBase,
-    ScreenManager, Display, Point, ObjectTransform, PropertyMap, NotificationOptions, ContainerNotification,
-    TrayIconDetails, MenuItem, Guid, MessageBus, MessageBusSubscription, MessageBusOptions, GlobalShortcutManager,
-    EventArgs, WindowEventArgs
+  registerContainer,
+  ContainerWindow,
+  PersistedWindowLayout,
+  Rectangle,
+  Container,
+  WebContainerBase,
+  ScreenManager,
+  Display,
+  Point,
+  ObjectTransform,
+  PropertyMap,
+  NotificationOptions,
+  ContainerNotification,
+  TrayIconDetails,
+  MenuItem,
+  Guid,
+  MessageBus,
+  MessageBusSubscription,
+  MessageBusOptions,
+  GlobalShortcutManager,
+  EventArgs,
+  WindowEventArgs,
```

Comment:
This is recommended?

Reply:
It is indeed. Another example: https://github.com/facebook/react/blob/9198a5cec0936a21a5ba194a22fcbac03eba5d1d/packages/react-devtools-scheduling-profiler/src/view-base/__tests__/geometry-test.js#L10
 _Note: you'd have to increase the print width to 390 to get it all on one line, it doesn't have an option to change it to multiple lines._

There is not an option to change this and thats kind of the point, maybe prettier will change how it formats imports in the future, I don't know, but the point of prettier is it is not to provide a kitchen sink of options, to satisfy an individual developers taste perfectly. But, to provide a consistent set of styling with very few options to avoid bike shedding. -https://prettier.io/docs/en/option-philosophy.html

If the core desktopJs team decides that is not the kind of tool you want to use then thats okay that I won't take it personally :)